File: LEDBanner/class/Controller/CustomSettingsViewController.m
//
//  CustomSettingsViewController.m
//  LEDBanner
//
//  Created by 何少博 on 16/12/29.
//  Copyright © 2016年 shaobo.He. All rights reserved.
//

#import "CustomSettingsViewController.h"
#import "UIColor+x.h"
@interface CustomSettingsViewController ()

@end

@implementation CustomSettingsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.tableView.separatorColor = color_303844;
    self.tableView.backgroundColor = color_000000;
    self.navigationController.navigationBar.hidden = NO;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (UIColor *)tableViewHeadViewBackgroundColor {
    return color_000000;//[UIColor colorWithRed:(231.0/255.0) green:(231.0/255.0)  blue:(231.0/255.0)  alpha:1.0];
}
- (UIColor *)tableViewHeadViewTextColor {
    return [UIColor lightGrayColor];
}
-(UIColor *)tableViewCellBackgroundColor{
    return color_131a20;
}
-(UIColor *)tableViewCellTextColor{
    return color_ffffff;
}
-(UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath{
    UITableViewCell * cell = [super tableView:tableView cellForRowAtIndexPath:indexPath];
    cell.detailTextLabel.textColor = color_ffffff;
    cell.selectedBackgroundView = [[UIView alloc]initWithFrame:cell.frame];
    cell.selectedBackgroundView.backgroundColor = color_000000;
    return cell;
}
/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: VideoCompress/Classs/CustomSettins/CustomSettingsViewController.swift
//
//  CustomSettingsViewController.swift
//  VideoCompress
//
//  Created by 何少博 on 16/12/4.
//  Copyright © 2016年 shaobo.He. All rights reserved.
//

import UIKit

class CustomSettingsViewController: SettingsViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let backImage = UIImage(named: "back")
        navigationItem.leftBarButtonItem = UIBarButtonItem(image: backImage, style: .plain, target: self, action: #selector(dismisSelf))
        let navibarImage = UIImage(named: "navigation_bar")
        let stratchedNavibarImage = navibarImage?.resizableImage(withCapInsets: UIEdgeInsets.zero, resizingMode: .stretch)
        navigationController?.navigationBar.setBackgroundImage(stratchedNavibarImage, for: .default)
        navigationController?.navigationBar.tintColor = UIColor.white
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func dismisSelf() {
        dismiss(animated: true, completion: nil)
    }
    
    override func tableViewCellTextColor() -> UIColor! {
        return UIColor.white
    }
    override func tableViewBackgroundColor() -> UIColor! {
        return back_Color
    }
    override func tableViewCellBackgroundColor() -> UIColor! {
        return back_Color
    }

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
